### DISCUSSION TITLE
COM-ports

### DISCUSSION THREAD

**Discussion by mezantrop** [jbruchon/elks#1773] [Q&A] [answered]
Hello dear ELKers, I'm seeking a "modern" OS for my old 80386SX laptop, and I'm pretty sure ELKS would be ideal for that. But, I want to put it on the net somehow. Unfortunately, the old machine has no ethernet, only an old and slow COM port exists. So, the question, can I use COM ports in ELKS for SLIP or PPP to achieve the goal? Sorry, if I'm blindly missed the info.

**Comment by ghaerr** [comment, marked as answer]
Hi @mezantrop, ELKS supports SLIP but not PPP, so you should be fine. When first starting, try with lower baud rates e.g 9600 as some older systems are quite slow and can lose UART characters. If required, we have some provisions (CONFIG_FAST_IRQ4) that will allow operation at 38400 or higher. The Wiki has information on how to setup networking.

**Reply by mezantrop** [reply]
Thank you very much @ghaerr, that's exactly what is required!
